plugin/js/showpass-custom.js

```diff
@@ -124,37 +124,91 @@
 				showpass.tickets.calendarWidget(id, params);
 			});
 
-			const embeddedCalendarExists = document.getElementById('showpass-calendar-widget');
-			if (embeddedCalendarExists) {
-				let script = document.createElement("script");
-				script.type = "text/javascript";
-				script.src = 'https://showpass.com/static/dist/sdk.js';
-
-				let useBeta = $('#option_use_showpass_beta').val();
-				let useDemo = $('#option_use_showpass_demo').val();
-				if (useBeta) {
-					script.src = 'https://beta.showpass.com/static/dist/sdk.js';
-				} else if (useDemo) {
-					script.src = 'https://demo.showpass.com/static/dist/sdk.js';
-				}
+			function initializeShowpassEmbeddedWidgets() {
```

Comment:
I just made the `embeddedCalendarExists` logic more generic so that it can handle different types of embedded widgets